Src/StartMenu/StartMenuDLL/MenuContainer.cpp

```diff
@@ -444,7 +444,10 @@ LRESULT CALLBACK CMenuContainer::SubclassSearchBox( HWND hWnd, UINT uMsg, WPARAM
 			SetBkColor(hdc,GetSysColor(COLOR_WINDOW));
 			SetBkMode(hdc,TRANSPARENT);
 			SetTextColor(hdc,s_Skin.Search_text_colors[1]);
-			DrawText(hdc,pParent->m_Items[pParent->m_SearchIndex].name,-1,&rc,DT_SINGLELINE|DT_EDITCONTROL|(s_bRTL?DT_RIGHT:DT_LEFT));
+			if (GetSettingBool(L"SearchHint"))
```

Comment:
It would be better to have local variable that will hold actual string.
And then call `DrawText` at one place (with proper string).
So that we don't need to duplicate all the parameters (and risk they will get out of sync in the future).